missingkids/less/css/application.css

```diff
@@ -8865,6 +8865,182 @@ textarea.form-control {
   height: 120px;
 }
 /* Blue-base text color */
-p.textColorBlueBase {
+.textColorBlueBase {
   color: #006C8A;
 }
+.textColorLightBlue {
+  color: #92D6E3;
+}
+.textarea {
+  width: 90%;
+  height: 300px;
+  margin: 16px;
+}
+.radio-section {
+  border: 1px solid red;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  display: inline-block;
+}
+.radio-section-individual {
+  border-right: 1px solid red;
+  display: inline-block;
+}
+.radio-section-individual label {
+  margin-bottom: 0px;
+  padding: 12px;
+}
+.form-horizontal-cover {
+  padding: 16px;
+  margin-bottom: 20px;
+  border: 1px solid grey;
+}
+.radio-section-individual:last-child {
+  border-right: none;
+}
+.form-horizontal-cover .form-horizontal .control-label {
+  text-align: left;
+}
+form.form-horizontal label {
+  text-align: left;
+}
+.radio-section input {
+  display: none;
+}
+/*input[type="radio"]:checked+div .radio-section-individual{
+     background-color: red;
+ }*/
+input[type="radio"]:checked + label:first-child {
```

Comment:
There are styles from the search for missing children widget. This will get applied to all the radio buttons in the site. please scope this to the respective module